### DISCUSSION TITLE
Error: Extension 'GitHub.copilot-chat' CANNOT use API proposal: aiTextSearchProvider.

### DISCUSSION THREAD

**Discussion by wuxiaohan2023** [community/community#121048] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have logged into Copilot but int output show this info ,   
Currently, only chat is available, but there is no automatic prompt in the code editing area
Can someone help me see what the problem is，

Tshank~

<img width="954" alt="image" src="https://github.com/community/community/assets/145582538/f442b722-fa96-4efe-8265-1dd3d0528e82">

**Comment by wuxiaohan2023** [comment]
Addendum: I am currently using vsCode

**Comment by buzcarter** [comment]
Same error in VSCode console. 

- VSCode version: 1.88.1 (Universal), 
- OS: Darwin x64 23.4.0 
- MacbookPro/Intel
- VS Code Speech v0.8.0
- GitHub Copilot Chat v0.14.1

**Reply by wuxiaohan2023** [reply]
first view  ping copilot-proxy.githubusercontent.com  result , if can't connect , view host config file.

**Reply by mcqj** [reply]
I'm seeing the error and I can connect
